frontend/src/scenes/saved-insights/savedInsightsLogic.test.ts

```diff
@@ -192,24 +192,26 @@ describe('savedInsightsLogic', () => {
     })
 
     it('can duplicate and does not use derived name for name', async () => {
-        const sourceInsight = createInsight(123, 'hello')
+        const sourceInsight = createInsight(123, 'hello') as QueryBasedInsightModel
         sourceInsight.name = ''
         sourceInsight.derived_name = 'should be copied'
         await logic.asyncActions.duplicateInsight(sourceInsight)
         expect(api.create).toHaveBeenCalledWith(
             `api/projects/${MOCK_TEAM_ID}/insights`,
-            expect.objectContaining({ name: '' })
+            expect.objectContaining({ name: '' }),
+            expect.objectContaining({})
```

Comment:
`ApiRequest.create` is implemented like this:

```
return await api.create(this.assembleFullUrl(), options?.data, options)
```

Meaning the data gets repeated in options. I didn't want to change the implementation in this PR, therefore adjusting tests.